cargo-afl/tests/integration.rs

```diff
@@ -46,3 +46,46 @@ fn integration() {
     }
     assert!(temp_dir_path.join("default").join("fuzzer_stats").is_file());
 }
+
+#[test]
+fn integration_cfg() {
+    let temp_dir = tempfile::TempDir::new().expect("Could not create temporary directory");
+    let temp_dir_path = temp_dir.path();
+
+    assert_cmd::Command::new(cargo_afl_path())
```

Comment:
Could we please put this and the code that follows in a loop over `[false, true]`, so that:
- in the first case, `AFL_NO_CFG_FUZZING` is not set and the test verifies that the program does not crash
- in the second case, `AFL_NO_CFG_FUZZING` is set and the test verifies that the program does crash (like it does now)?

Reply:
Done!